content/docs/welcome-guide/supported-platforms.md

```diff
@@ -0,0 +1,32 @@
+---
+linktitle: Supported Platforms
+title: Supported Platforms in Open 3D Engine
+description: Get a list of host platforms and target platforms supported by Open 3D Engine (O3DE).
+weight: 250
+toc: true
+---
+
+**Open 3D Engine (O3DE)** supports several _host platforms_ for creating and developing games and simulations, and _target platforms_ on which these projects can run.
+
+### Host platforms
+
+| Host Platform | Platform-specific Documentation |
+| --- | --- |
+| Microsoft Windows | <ul><li>[Windows prerequisites and configuration](requirements/#microsoft-windows)</li><li>[Installing O3DE for Windows](setup/installing-windows)</li><li>[Creating projects from the CLI on Windows](create/creating-projects-using-cli/creating-windows/)<sup>1</sup></li><li>[Building O3DE from source on Windows](setup/setup-from-github/building-windows/)</li></ul> |
+| Linux | <ul><li>[Linux prerequisites and configuration](requirements/#linux)</li><li>[Installing O3DE for Linux](setup/installing-linux)</li><li>[Creating projects from the CLI on Linux](create/creating-projects-using-cli/creating-linux/)<sup>1</sup></li><li>[Building O3DE from source on Linux](setup/setup-from-github/building-linux/)</li></ul> |
+| macOS | Support for developing on **macOS** is in an experimental stage. Documentation coming in 2022. |
+
+<sup>1</sup> Platform-independent documentation on creating projects using **Project Manager** can be found [here](create/creating-projects-using-project-manager).
+
+For minimum hardware requirements for development on host platforms, refer to [System requirements](requirements).
```

Comment:
```suggestion
For minimum hardware requirements for development on host platforms, refer to [System Requirements](requirements).
```

Reply:
Funny you mention that, I kind of prefer this capitalization on links too, even though I've seen editors change this to use sentence casing.